app.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ah-frontend-space",
+    "scripts": {},
+    "env": {},
+    "formation": {
+      "web": {
+        "quantity": 1
+      }
+    },
+    "addons": [],
+    "buildpacks": [
+      {
+        "url": "heroku/nodejs"
+      },
+      {
+        "url": "https://github.com/heroku/heroku-buildpack-static"
+      }
+    ],
+    "stack": "heroku-18"
+  } 
+  
```

Comment:
@sebbss , kindly add a new line to this file as well as to the `statis.json` file. Other than that, I have tested the deployed app and it works as described in the PR.